README.rst

```diff
@@ -404,6 +404,7 @@ Variables you might want to override
 * ``PKG`` is where package definitions are searched. When building packages,
   each ``*.pkg`` file in that directory will be built. By default ``$T/pkg``.
 * ``PKG_DEFAULTS`` contains the default options passed to ``mkpkg``.
+* ``PKG_FILES`` contains the list of packages definitions.
```

Comment:
I would add a bit more than this in the Packaging section. Like the example in the release notes. Also I would mention the `pkg` variable as a more general approach to add stuff to do for the `make pkg` target.